composer.json

```diff
@@ -6,11 +6,18 @@
   "license": "GPL-3.0-or-later",
   "prefer-stable": true,
   "minimum-stability": "dev",
-  "require": {},
+  "require": {
+    "php": ">=5.3|>=7.0"
```

Comment:
5.3 is not the correct version.
`AS_DBLogger`, `BatchFether`, `AS_DBStore` use  short array `[]` syntax, which is PHP 5.4+.

Reply:
Also, don't forget about the Migration classes, there is PHP 5.5 checks there.

Reply:
> AS_DBLogger, BatchFether, AS_DBStore use short array [] syntax, which is PHP 5.4+.

The migration classes and tables data stores are only loaded if the `MIN_PHP_VERSION` is met. AS runs in deprecated mode using the post store on 5.3-4. The automated unit testing still runs against 5.3 & 5.4.

Reply:
So if I want to use custom tables to store the queue I'm required to be on PHP 5.5+, correct? On older PHP everything will be stored in wp_posts table.

Reply:
@slaFFik yes that's correct.

Reply:
@thenbrent Does the lib handles the migration of a live site from `wp_posts` to custom tables when PHP version is changed on a server from 5.4 to 5.5?

Reply:
Does the lib handles the migration of a live site from wp_posts to custom tables when PHP version is changed on a server from 5.4 to 5.5?

@slaFFik Thanks for your questions. You highlighted that we needed to add some migration documentation. I created #424 with a v3.0 specific FAQ.